tests/integration/test_numeric_array.py

```diff
@@ -0,0 +1,40 @@
+# encoding: utf-8
+
+"""Integration-test suite for numeric-arrays."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+
+from cr.cube.cube import Cube
+
+from ..fixtures import NA
+
+
+# pylint: disable=no-self-use
+class TestNumericArrays:
+    """Test-suite for numeric-arrays behaviors."""
+
+    def test_num_arr_x_cat_with_missings(self):
+        """Test means on numeric array, grouped by single categorical dimension."""
+        slice_ = Cube(NA.NUM_ARR_X_CAT_WITH_MISSINGS).partitions[0]
```

Comment:
The name of the fixture wrongly implies that we have a cross product of something (e.g. in the dimensions), when there's none. A better name would be `NUM_ARR_MEANS_GROUPED_BY_CAT`. So we know exactly what kind of cube we're dealing with.